### PR TITLE
Fw 3524 accessibility menus

### PR DESCRIPTION
**Pull request by Cara-Barter**
#### Definition of Done
- [x] Acceptance criteria has been met

### CODE REVIEW

File: frontend/app_v2/src/components/NavBar/NavBarPresentationMobile.js [hunk 1]
```diff
@@ -90,10 +90,10 @@ function NavBarPresentationMobile({ menuData, sitename }) {
               <a
                 type="button"
                 onClick={() => setIsSubMenuOpen(false)}
-                className="w-full m-3 p-1 text-primary flex items-center hover:bg-gray-50 focus:outline-none"
+                className="w-full m-3 p-1 text-primary flex items-center hover:bg-primary hover:text-white focus:outline-none"
               >
                 {useIcon('ChevronLeft', 'fill-current h-12 w-8')}
-                <span className="ml-3 text-primary font-medium hover:text-primary-dark">Back</span>
+                <span className="ml-3 text-primary font-medium hover:text-white">Back</span>
```
Comment (sarahfirstvoices) [inline frontend/app_v2/src/components/NavBar/NavBarPresentationMobile.js:96]: Should this have the text-primary and hover:text-xx classes removed, like the rest of the spans? I haven't rendered this-- if it's necessary just ignore me

File: frontend/app_v2/src/components/NavBar/NavBarPresentationMobile.js [hunk 2]
```diff
@@ -90,10 +90,10 @@ function NavBarPresentationMobile({ menuData, sitename }) {
               <a
                 type="button"
                 onClick={() => setIsSubMenuOpen(false)}
-                className="w-full m-3 p-1 text-primary flex items-center hover:bg-gray-50 focus:outline-none"
+                className="w-full m-3 p-1 text-primary flex items-center hover:bg-primary hover:text-white focus:outline-none"
```
Comment (sarahfirstvoices) [inline frontend/app_v2/src/components/NavBar/NavBarPresentationMobile.js:93]: Does this also need a focus ring?

File: frontend/app_v2/src/components/NavBar/NavBarPresentationMobile.js (hunk 1 above)
Comment (Cara-Barter) [reply frontend/app_v2/src/components/NavBar/NavBarPresentationMobile.js:96]: no, they still need to be there as it is rendered individually

File: frontend/app_v2/src/components/NavBar/NavBarPresentationMobile.js (hunk 2 above)
Comment (Cara-Barter) [reply frontend/app_v2/src/components/NavBar/NavBarPresentationMobile.js:93]: yes, good catch.  It's weird that the ring was showing up when I rendered it.  Must have been inheriting it from a parent

**Review by sarahfirstvoices — APPROVED** [review]

**Pull request merged**